nengo/dists.py

```diff
@@ -99,9 +102,9 @@ class Uniform(Distribution):
         If true, sample from a uniform distribution of integers. In this case,
         low and high should be integers.
     """
-    low = NumberParam()
-    high = NumberParam()
-    integer = BoolParam()
+    low = NumberParam('low')
```

Comment:
I agree, that this is not the nicest syntax, but I think it is ok in this place:

* These parameter definitions are not exactly what most users will do on a regular basis.
* Both `name`s are in the same line. So it is hopefully easy to spot inconsistent names.
* There are other places in Python that follow the same style, namely `namedtuple` or dynamically defining a class with `type`.